test/progress.js

```diff
@@ -26,43 +26,81 @@ test('report progress', async t => {
 	const buffer = crypto.randomBytes(THREE_HUNDRED_KILO);
 	fs.writeFileSync(t.context.source, buffer);
 
-	let callCount = 0;
-	await cpFile(t.context.source, t.context.destination).on('progress', progress => {
-		callCount++;
+	const progressHandler = progress => {
 		t.is(typeof progress.sourcePath, 'string');
 		t.is(typeof progress.destinationPath, 'string');
 		t.is(typeof progress.size, 'number');
 		t.is(typeof progress.writtenBytes, 'number');
 		t.is(typeof progress.percent, 'number');
 		t.is(progress.size, THREE_HUNDRED_KILO);
+	};
+
+	let callCountEvent = 0;
```

Comment:
I think `callCountEvent` would not be a good name than `callCount` since it is not a kind of `Event` object.